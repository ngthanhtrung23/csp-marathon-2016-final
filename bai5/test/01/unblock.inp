111..2
..3..2
003..2
4.3.55
4...6.
777.6.
