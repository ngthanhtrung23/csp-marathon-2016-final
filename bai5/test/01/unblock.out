15
5 L 1
2 D 1
1 R 3
3 U 1
5 L 1
6 U 1
7 R 3
4 D 1
5 L 2
3 D 3
0 R 3
3 U 1
7 L 1
2 D 2
0 R 1

